### PR TITLE
Remove roon join and unjoin services 

### PR DESCRIPTION
**Pull request by pavoni**
## Proposed change

This PR updates the roon documentation to remove the roon specific services that will be replaced  by use of the media_payer based services.

It also revises the documentation about installation since Roon now supports discovery (I must have missed this when I added that feature)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/49667
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/roon.markdown [hunk 1]
```diff
@@ -15,16 +15,11 @@ The Roon integration allows you to control [RoonLabs](https://roonlabs.com/) mus
...
-To integrate with Roon, you need to provide Home Assistant with the Hostname or IP address of the machine that runs your Roon Core, and then authorize Home Assistant in the Roon application.
-
-If you use an IP address, please assign a static IP address to the machine that runs Roon Core. This ensures that it won't change IP addresses, so you won't have to change the configuration in Home Assistant if it reboots and changes IP address. See your router's manual for details on how to set this up.
-
 ## Configuration
 
-You need the Hostname or IP address of the machine that runs your Roon Core. This might be a machine name (which can be followed by `.local`, e.g., `myserver.local`) or can be an IP address.
-
 1. From the Home Assistant front-end, navigate to **Configuration** then **Integrations**. Under **Set up a new integration** locate 'Roon' and click **Configure**.
-2. Enter the `Hostname` or `IP address` for the Roon Core machine and click **Submit**.
```
Comment (frenck) [inline source/_integrations/roon.markdown:27]: These changes are not related to the parent PR.
If anything, this whole block should be replaced by including the `integrations/config_flow.md` in a separate PR.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/roon.markdown (hunk 1 above)
Comment (pavoni) [reply source/_integrations/roon.markdown:27]: I've raised a new PR against current for the discovery stuff:  https://github.com/home-assistant/home-assistant.io/pull/17599

There is a roon specific authorisation step in the middle - so I couldn't use the include.

Will removes that stuff from this PR.

Comment (pavoni) [reply source/_integrations/roon.markdown:27]: Will close this and replace with https://github.com/home-assistant/home-assistant.io/pull/17600

Comment (frenck) [reply source/_integrations/roon.markdown:27]: That specific step in the middle should not be needed to be documented, as it should be clear what one should do from the UI directly.
